infra/recipes/terraform/azure/sandbox/mysql-services.tf

```diff
@@ -0,0 +1,33 @@
+resource "azurerm_subnet" "mysql_subnet" {
```

Comment:
This one too, I believe this can also be abstracted into top-level infra/recipes/terraform/azure/sandbox